src/platform/Zephyr/BUILD.gn

```diff
@@ -66,6 +66,13 @@ static_library("Zephyr") {
     }
   }
 
+if (chip_enable_wifi) {
```

Comment:
I think we shouldn't build these files for base Zephyr platform (not NCS) as we use NCS-specific APIs in the implementation.

Reply:
I was wondering if this should be here. Ok, will revert it.